carbonmark-api/src/utils/getCategoryFromMethodoloy.ts

```diff
@@ -0,0 +1,185 @@
+// import { CategoryName } from "../../../lib/types/carbonmark.types";
+// @todo correct types and import from lib if possible
+const getMap = () => {
+  const methodologyToCategoryMap = new Map<string, any>();
+
+  //Renewable Energy
+  methodologyToCategoryMap.set("AM0103", "Renewable Energy");
```

Comment:
Maybe not now but all these could be stored in the CMS for instance. So these could be updated without changing the code.

Reply:
definitely agree with this

Reply:
refactored to use project cms data and removed